### PR TITLE
changing oc get pods command to watch pods

### PR DESCRIPTION
**Pull request by samueltauil**

### CODE REVIEW

**Comment by zregvart** [comment]
I was wondering about that, I'm sure that @jimmidyson and @chirino know about `-w` switch, but perhaps wanted a nicer UI?

**Comment by samueltauil** [comment]
just added the `-w` switch because it can run across different os platforms as it is part of `oc` binary.

**Comment by zregvart** [comment]
@samueltauil to me that makes absolute sense, I just wandered why @jimmidyson and @chirino thought this was better 😸

**Comment by chirino** [comment]
Lgtm

**Comment by rhuss** [comment]
personally I prefer plain 'watch' as it is more dynamic and cleans up the history stuff. What's about adding both examples ?

**Comment by jimmidyson** [comment]
`-w` is a bit hard to follow imo, hence why i preferred `watch`. Happy to have both as examples, but :-1: to just `-w`

File: content/quickstart.md [hunk 1]
```diff
@@ -102,9 +102,15 @@ $ oc new-app syndesis-dev-restricted \
...
-Wait until all pods are running:
+Wait until all pods are running. You can either use OpenShift intrinsic watch feature for a line-by-line update
 
 ```bash
+$ oc get pods -w
+```
+
+or use `watch` for a more curses like full screen user interface:
+
+```
```
Comment (zregvart) [inline content/quickstart.md]: should this be

    ```bash
